File: src/main/java/no/nav/sbl/consumers/axsys/domain/tilgang/Enhet.java
package no.nav.sbl.consumers.axsys.domain.tilgang;

import lombok.*;

import java.io.Serializable;
import java.util.Set;

@Getter
@Setter
@EqualsAndHashCode
@AllArgsConstructor
@NoArgsConstructor
public class Enhet implements Serializable {
    private String enhetId;
    private Set<String> fagomrader;
    private String navn;
}

File: src/main/java/no/nav/sbl/consumers/norg2/domain/Enhet.kt
package no.nav.sbl.consumers.norg2.domain;

import lombok.*;

import java.io.Serializable;

@Getter
@Setter
@EqualsAndHashCode
@AllArgsConstructor
@NoArgsConstructor
public class Enhet implements Serializable {
    private String enhetNr;
    private String navn;
    private String status;
}
